aesara/configdefaults.py

```diff
@@ -378,6 +378,13 @@ def add_basic_configvars():
         in_c_key=False,
     )
 
+    config.add(
+        "DISABLE_NUMBA_CACHE",
```

Comment:
We might want to change this to something like `DISABLE_NUMBA_PYTHON_IR_CACHING`.